README.md

```diff
@@ -487,27 +769,151 @@ dlv attach --continue --accept-multiclient --api-version 2 --headless --listen :
+"https://${API_URI}/o2ims-infrastructureInventory/v1/deploymentManagers?fields=name" | jq
+```
+
+#### <a name='GETdeploymentManagersListusingfilter'></a>GET deploymentManagers List using filter
+
+To get a list of all the deploymentManagers whose name is **not** local-cluster in our O-Cloud:
+
+```bash
+$ curl --insecure --silent --header "Authorization: Bearer $MY_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/deploymentManagers?filter=(neq,name,local-cluster)" | jq
+ | jq
+```
+
+### <a name='QuerytheResourceserver'></a>Query the Resource server
+
+The resource server exposes endpoints for retrieving resource types, resource pools and resources objects. The server relies on the Search Query API of ACM hub. Follow these [instructions](docs/dev/env_acm.md#search-query-api) to enable
+and configure the search API access. The resource server will translate those REST requests and send them to the ACM search server that implements a graphQL API.
+
+> :exclamation: To obtain the requested information we need to enable the searchCollector of all the managed clusters, concretely, in the KlusterletAddonConfig CR.
+
+#### <a name='GETResourceTypeList'></a>GET Resource Type List
+
+To get a list of available resource types:
+
+```bash
+$ curl -ks --header "Authorization: Bearer $MY_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/resourceTypes" | jq
+```
+#### <a name='GETSpecificResourceType'></a>GET Specific Resource Type 
+
+To get information of a specific resource type:
+
+```bash
+$ curl -ks --header "Authorization: Bearer $MY_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/resourceTypes/${resource_type_name} | jq
+```
+
+#### <a name='GETResourcePoolList'></a>GET Resource Pool List
+
+To get a list of available resource pools:
+
+```bash
+$ curl -ks --header "Authorization: Bearer $MY_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/resourcePools" | jq
+```
+
+#### <a name='GETSpecificResourcePool'></a>GET Specific Resource Pool
+
+To get information of a specific resource pool:
+
+```bash
+$ curl -ks --header "Authorization: Bearer $MY_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/resourcePools/{resourcePoolId}" | jq
+```
+
+#### <a name='GETallResourcesofaspecificResourcePool'></a>GET all Resources of a specific Resource Pool
+
+We can filter down to get all the resources of a specific resourcePool.
+
+```bash
+$ curl -ks --header "Authorization: Bearer $MY_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/resourcePools/{resourcePoolId}
+/resources" | jq
+```
+
+### <a name='QuerytheInfrastructureInventorySubscriptionResourceServer'></a>Query the Infrastructure Inventory Subscription (Resource Server)
+
+#### <a name='GETInfrastructureInventorySubscriptionList'></a>GET Infrastructure Inventory Subscription List
 
 To get a list of resource subscriptions:
 
 ```
 $ curl -s http://localhost:8004/o2ims-infrastructureInventory/v1/subscriptions | jq
```

Comment:
These next few curl commands also need the `Authorization` header... sorry, my mistake for missing that when I moved these lines.